spec/rails/class_loading_system_spec.rb

```diff
@@ -33,11 +36,29 @@
   end
 
   it "should not allow me to define widgets outside of app/views/" do
-    expect_exception('widget_defined_outside_app_views', 'ActionView::MissingTemplate', /class_loading_system_spec\/widget_defined_outside_app_views/)
+    # TODO: what is this spec testing? the action doesn't tell rails to render the outside widget
```

Comment:
im guessing this spec is testing that rails/fortitude doesn't autoload and use a widget defined outside of app/views, but im not 100% sure. Will investigate this along with the rest of the autoloading questions